restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/ConferencesEndpoint.java

```diff
@@ -194,4 +208,77 @@ protected Response getConferences(final String accountSid, UriInfo info, MediaTy
         }
     }
 
+    protected Response updateConference(String accountSid, String sid, MultivaluedMap<String, String> data,
+            MediaType responseType) {
+        logger.info(String.format("updateConference accountsid: %s conferenceSid: %s", accountSid, sid));
+        Account account = daoManager.getAccountsDao().getAccount(accountSid);
+        try {
+            secure(account, "RestComm:Read:Conferences");
```

Comment:
is this the correct literal "RestComm:Read:Conferences"? i dont know the convention, but this is not a ReadOnly operation, here we are modifying the conference, shouldnt be somethign like "Restcomm:Write:Conferences"?

Reply:
nice catch, updated